test/test.php

```diff
@@ -391,6 +391,30 @@ function () use ($router) {
     }
 );
 
+test(
```

Comment:
This test wasn't unused - there was a bug in `mindplay/testies` which caused tests with duplicate names to override each other. I've fixed this in `mindplay/testies^0.3.1`.